packages/shared/lib/core/network/constants/default-network-metadata.constant.ts

```diff
@@ -1,83 +1,37 @@
-import { COIN_TYPE, DEFAULT_BASE_TOKEN } from '../constants'
-import { NetworkId, TokenSupply } from '../enums'
+import { COIN_TYPE } from '../constants'
+import { NetworkId } from '../enums'
 import { IStardustNetworkMetadata } from '../interfaces'
 import { NetworkMetadata } from '../types'
+import { nodeInfoBaseToken, nodeInfoProtocol } from '../stores/node-info.store'
+import { get } from 'svelte/store'
 
 export const DEFAULT_NETWORK_METADATA: Readonly<{ [key in NetworkId]?: NetworkMetadata }> = {
     [NetworkId.Shimmer]: <IStardustNetworkMetadata>{
         id: NetworkId.Shimmer,
         name: 'Shimmer',
         coinType: COIN_TYPE[NetworkId.Shimmer],
-        protocol: {
-            version: 2,
-            networkName: 'shimmer',
-            bech32Hrp: 'smr',
-            minPowScore: 1500,
-            belowMaxDepth: 15,
-            rentStructure: {
-                vByteCost: 100,
-                vByteFactorData: 1,
-                vByteFactorKey: 10,
-            },
-            tokenSupply: TokenSupply.Shimmer,
-        },
-        baseToken: DEFAULT_BASE_TOKEN[NetworkId.Shimmer],
+        protocol: get(nodeInfoProtocol),
```

Comment:
This is basically undefined, so all non-custom profiles in onboarding won't have neither `protocol` or `baseToken`, instead, these two will be added by `checkAndUpdateActiveProfileNetwork`. I don't think this is a good idea

Reply:
Are `protocol` or `baseToken` needed in onboarding?
In onboarding, the `id` and `name` of the `network` are used, but neither the `protocol` nor the `basetoken` is used I think

Reply:
if they are not being used then the type is wrong

Reply:
ready ✅